expenses/migrations/0003_alter_expenses_date.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-11 19:56
```

Comment:
I think @Yoavsc129 solves that problem, Please use it and don't create a new alter file

Reply:
Done